.github/workflows/clj-build.yml

```diff
@@ -0,0 +1,142 @@
+          echo "sha=${SHA}" >> $GITHUB_OUTPUT
+          SHORT_SHA=$( git rev-parse --short HEAD )
+          echo "SHORT_SHA=${SHORT_SHA}" >> $GITHUB_ENV
+          echo "short_sha=${SHORT_SHA}" >> $GITHUB_OUTPUT
+          VERSION=$( date -u '+%Y%m%d' )-${{ github.run_number || 0 }}-$( git rev-parse --short HEAD )
+          echo "VERSION=${VERSION}" >> $GITHUB_ENV
+          echo "version=${VERSION}" >> $GITHUB_OUTPUT
+
+      - name: download artifacts
+        uses: actions/download-artifact@v3
+
+      - name: auth with docker
+        run: |
+          echo ${{ secrets.GHCR_USER_PAT }} | docker login ghcr.io -u ${{ secrets.GHCR_USER_NAME }} --password-stdin
+
+      - name: build release image
+        run: |
+          docker build \
+            -t ${{ steps.vars.outputs.image }}:${{ steps.vars.outputs.branch_tag }} \
+            -t ${{ steps.vars.outputs.image }}:${{ steps.vars.outputs.version }} \
+            --build-arg BUILD_NUMBER=${{ github.run_number }} \
+            --build-arg BUILD_SHA=${{ steps.vars.outputs.sha }} \
+            --build-arg RP_PRIVATE_TOKEN=${{ secrets.GPR_TOKEN }} \
+            --build-arg VERSION=${{ steps.vars.outputs.version }} \
+            --cache-from ${{ steps.vars.outputs.image }}:${{ steps.vars.outputs.branch_tag }} \
+            --target release \
+            -f Dockerfile .
+
+      - name: push to docker registry
+        run: |
+          docker push ${{ steps.vars.outputs.image }}:${{ steps.vars.outputs.branch_tag }}
+          docker push ${{ steps.vars.outputs.image }}:${{ steps.vars.outputs.version }}
+          # reserve the `latest` tag for the target branch
+          # branch-tags are essentially `latest` per branch
+          if [[ ${{ contains(github.ref, inputs.target_branch) }} = true ]]; then
+            docker tag ${{ steps.vars.outputs.image }}:${{ steps.vars.outputs.branch_tag }} ${{ steps.vars.outputs.image }}:latest
+            docker push ${{ steps.vars.outputs.image }}:latest
+          fi
+
+      - name: Add build tag check
+        uses: LouisBrunner/checks-action@v1.6.0
+        if: github.event.pull_request
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+          name: |
+            Image tag: ${{ steps.vars.outputs.version }}
+          conclusion: ${{ job.status }}
+
+      - name: git tag
+        if: contains(github.ref, 'refs/heads/${{ inputs.target_branch }}')
+        shell: bash {0}
+        run: |
+          git config --global user.email "idg@rentpath.com"
+          git config --global user.name "rentpath-plateng"
+          git tag -a ${{ steps.vars.outputs.version }} -m "${{ steps.vars.outputs.version }}" && \
+            git push origin refs/tags/${{ steps.vars.outputs.version }}:refs/tags/${{ steps.vars.outputs.version }}
+
+      - name: release
+        if: contains(github.ref, 'refs/heads/${{ inputs.target_branch }}')
+        id: create_release
+        uses: actions/create-release@v1
```

Comment:
This version has been archived and contains deprecated commands. The [related repo](https://github.com/actions/create-release) provides some alternatives (I saw the [elixir team uses](https://github.com/rentpath/actions-services-workflows/blob/main/.github/workflows/deployment-images.yaml#L133) `softprops/action-gh-release@v1` - might need to update the _with_ parameters.